dotcom-rendering/src/web/components/LiveBlock.tsx

```diff
@@ -62,6 +63,7 @@ export const LiveBlock = ({
 					index,
 					pageId,
 					webTitle,
+					supportsDarkMode: true,
```

Comment:
I suspect you want this to be `false`? Dotcom doesn't support dark mode yet.

Reply:
See above